src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="requires">
+
+      <div class="table">
+        <fieldset>
+
+          <legend>Personal information</legend>
+
+          <div class="Surname" minlenght="5"
+        maxlenght="32">
+       Surname:
+            <input type="text">
+          </div>
```

Comment:
```suggestion
          <label class="Surname">
            Surname:
            <input
              type="text"
              minlenght="5"
              maxlenght="32"
            >
          </label>
```
all your layout should look like this